src/browser/provider/built-in/index.js

```diff
@@ -0,0 +1,14 @@
+import PathBrowserProvider from './path';
+import LocalBrowserProvider from './local';
+import RemoteBrowserProvider from './remote';
+import saucelabsBrowserProvider from './saucelabs';
+import phantomJSBrowserProvider from './phantomjs';
+
+
+export default {
+    'local':     new LocalBrowserProvider(),
+    'path':      new PathBrowserProvider(),
```

Comment:
> We can replace with localIy-installed

so, the `path` provider is purely for portable browsers and makes no sense if a browser is installed? what about exotic browsers?